tests/test_operation_definition.py

```diff
@@ -32,6 +40,11 @@ def test_get_definition():
         regions=[("region", RegionDef())])
 
 
+################################################################################
+#                            Invalid definitions                               #
```

Comment:
I would avoid these styles of docstrings, but up to what you guys like better, I would prefer something that renders as code-area in an editor...like a class that groups some tests